pkg/compact/compact.go

```diff
@@ -440,23 +442,23 @@ func (cg *Group) Resolution() int64 {
 
 // Compact plans and runs a single compaction against the group. The compacted result
 // is uploaded into the bucket the blocks were retrieved from.
-func (cg *Group) Compact(ctx context.Context, dir string, comp tsdb.Compactor) (ulid.ULID, error) {
+func (cg *Group) Compact(ctx context.Context, dir string, comp tsdb.Compactor) (bool, error) {
```

Comment:
Would be good to add to comment what the bool represents. `Will returns true if you should continue compaction on subsequent groups.`